--- mongo-cxx-driver.spec.orig	2021-09-07 10:45:09.609476580 -0400
+++ mongo-cxx-driver.spec	2021-09-07 10:48:52.781980206 -0400
@@ -1,8 +1,10 @@
 # for better compatibility with SCL spec file
 %global pkg_name mongo-cxx-driver
 
+%define _unpackaged_files_terminate_build 0
+
 Name:           mongo-cxx-driver
-Version:        3.6.5
+Version:        3.10.2
 Release:        1%{?dist}
 Summary:        A C++ Driver for MongoDB
 License:        ASL 2.0
@@ -10,8 +12,8 @@
 Source0:        https://github.com/mongodb/%{pkg_name}/archive/%{name}-r%{version}.tar.gz
 
 
-Patch2:         mongo-cxx-driver-3.3.1_paths.patch
-Patch3:         mongo-cxx-driver-catch-update.patch
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(libutf8proc)
 BuildRequires:  boost-devel >= 1.49
 BuildRequires:  openssl-devel
 BuildRequires:  cmake
@@ -67,8 +67,6 @@
 %prep
 %setup -q -n %{name}-r%{version}
 
-%patch2 -p1 -b .paths
-%patch3 -p1 -b .catchupdate
 
 
 %build
@@ -79,6 +77,9 @@
 %cmake \
     -DCMAKE_BUILD_TYPE=Release \
     -DBSONCXX_POLY_USE_BOOST=1 \
+    -DBUILD_VERSION=3.10.2 \
+    -DENABLE_UNINSTALL=OFF \
+    -DENABLE_TESTS=OFF \
     .
 
 %changelog
 * Mon Aug 09 2021 Honza Horak <hhorak@redhat.com> - 3.6.5-1
